thinkhazard/static/less/report.less

```diff
@@ -252,6 +252,21 @@ body {
   }
 }
 
+footer {
+  display: table;
+  white-space: nowrap;
+  &:before, &:after {
+    border-top: 5px solid #e6e6e6;
+    content: '';
+    display: table-cell;
+    position: relative;
+    top:0.5em ;
```

Comment:
This looks good to me in my browsers (Chrome and Firefox).

![screenshot from 2015-07-23 13 25 38](https://cloud.githubusercontent.com/assets/319774/8849238/5ff794b8-313e-11e5-80b7-b3552b3a2121.png)